presto-geospatial-toolkit/src/main/java/com/facebook/presto/geospatial/JtsGeometryUtils.java

```diff
@@ -181,6 +301,63 @@ private static Geometry readPolyline(SliceInput input)
         return GEOMETRY_FACTORY.createMultiLineString(lineStrings);
     }
 
+    /**
+     * Serialize a geometry to an ESRI polyline.
```

Comment:
This method doesn't need a comment. If you like to keep it, let's cleanup it a bit. How about this:

```
Serialize a LineString or MultiLineString to an ESRI Polyline.
```

Then, drop "Note:..." and @param.